### DISCUSSION TITLE
Any updates on pc deploy?

### DISCUSSION THREAD

**Discussion by simonmesmith** [reflex-dev/reflex#445] [General]
Now that I've been playing with Pynecone, I have lots of ideas for things I'd love to implement. 

Unfortunately, in my experience, deploying Flask-based apps with SQL databases (e.g. Google Cloud Run with Cloud SQL) is relatively too painful to allow for a lot of creative experimentation.

So I was wondering if there were any updates on pc deploy and any information on what hosting might entail.

PS: Congrats on YC backing!

**Comment by Alek99** [comment]
Thank you! We are working on this right now. We have alpha of our hosting service which we are using to host our main website and example apps. Working on making it secure and scalable before we role it our publicly. I can't give a specific date yet, but we are hoping to start onboard users sometime in March.

We may have a small beta group of users which we invite to test out the hosting service early. Let me know if you are interested in being apart of that.

**Reply by simonmesmith** [reply]
Great! If I have anything worth deploying by March, I'd be interested in being a beta tester 🙂

**Reply by duttaoindril** [reply]
Very interested!
